powersimdata/input/grid.py

```diff
@@ -63,13 +65,15 @@ def __init__(self, interconnect, source="usa_tamu", **kwargs):
         self.storage = network.storage
         self.grid_model = network.grid_model
         self.model_immutables = network.model_immutables
+        self.version = version
```

Comment:
There is a small issue  - when we have a cached network, the version will be set to `None` even for europe networks. I think we can fix that by removing the `version = network.version` on L#46 and setting `self.version = getattr(network, "version", None)` here.

Reply:
Done